test/honeycomb-options.test.ts

```diff
@@ -334,7 +335,7 @@ describe('metrics endpoint', () => {
   it('does not append path for grpc exporter protocol', () => {
     const options = computeOptions({
       metricsEndpoint: 'my-custom-endpoint',
-      protocol: 'grpc',
+      protocol: OtlpProtocolKind.Grpc,
```

Comment:
What do you think about also updating the test on "protocol" around line 472, to something like 'uses provided protocol option if set as string', and add test for 'uses provided protocol option if set as enum'.

```js
  it('uses provided protocol option if set as enum', () => {
    const options = computeOptions({
      protocol: OtlpProtocolKind.Grpc,
    });
    expect(options.protocol).toBe('grpc');
  });
```